components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -206,19 +206,35 @@ def _split_channel_info(rchannels):
     channels = []
     windows = []
     colors = []
+    reverses = []
     for chan in rchannels.split(','):
+        # chan  1|12:1386r$0000FF
         chan = chan.split('|', 1)
+        # chan ['1', '12:1386r$0000FF']
         t = chan[0].strip()
+        # t = '1'
         color = None
+        rev = None
+        # Not normally used...
         if t.find('$') >= 0:
             t, color = t.split('$')
         try:
             channels.append(int(t))
             ch_window = (None, None)
             if len(chan) > 1:
                 t = chan[1].strip()
+                # t = '12:1386r$0000FF'
                 if t.find('$') >= 0:
                     t, color = t.split('$', 1)
+                    # color = '0000FF'
+                    # t = 12:1386r
+                # Optional flag to enable/disable reverse codomain
+                if t.endswith('-r'):
```

Comment:
@will-moore sorry, I know I'm being a bit knitpicky here but would it be a major problem if we handled that with a split for ':' . Syntactically it seems a tad clearer/cleaner to get a tuple/triple with min:max:optRev.